app/api/responseCodes.js

```diff
@@ -0,0 +1,3 @@
+export const RESPONSE_CODES = {
+  SUCCESS: 200,
```

Comment:
NOTE - Just putting this file in place for now. This will be completely modified later to consist of multiple sections corresponding to each endpoint and its corresponding codes.